website/app/domain/contracts/content.ts

```diff
@@ -0,0 +1,86 @@
+export type Event = {
```

Comment:
well, it depends. Contracts are the interface on which any app part can rely, it seems that you talk about implementation with Pocketbase.

in a perfect world pocketbase is just an implementation of a storage to get the content you want.
with maybe some mapper in the middle.

Reply:
Then, I would have a generic `modules/storage` that wraps `modules/pocketbase` and only use `storage` across the app? I like the "everything is a module" because, like with components, it allows you to build anything!